src/components/POLine/POLineForm.js

```diff
@@ -263,7 +270,9 @@ class POLineForm extends Component {
                         label={<FormattedMessage id="ui-orders.line.accordion.eresource" />}
                         id={ACCORDION_ID.eresources}
                       >
-                        <EresourcesForm {...this.props} />
+                        <EresourcesForm
+                          {...this.props}
```

Comment:
basically, here is the same anti-pattern - passing down all the props, no matter if they are required or not. So if you touch this place then it's better to remove the code smell

Reply:
Updated